bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Teaser.java

```diff
@@ -36,71 +36,93 @@
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_WITH_CTA = "withCTA";
+    String PN_ACTIONS_ENABLED = "actionsEnabled";
 
     /**
-     * Name of the parent node where the Call-to-Action elements are stored
+     * Name of the child node where the Call-to-Action elements are stored
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String NN_CTAS = "ctas";
+    String NN_ACTIONS = "actions";
 
     /**
-     * Name of the property that will store if the image link should be hidden.
+     * Name of the resource property that stores the Call-to-Action link
```

Comment:
I'd remove `resource`

Reply:
I was a bit confused as we are sometimes mixing Node and Resource terminology: e.g. `node name` and here `resource property`. Maybe we should start using `page property` as opposed to `policy property`?